Node.gitignore

```diff
@@ -5,6 +5,9 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+# Diagnostic reports
+report.*.json
```

Comment:
I'm a bit nervous about this rule catching other files that happen match this pattern (by luck or sheer misfortune) - is there a more strict way we could write this expression?

I see from the docs that the file paths look like this: `report.20181221.005011.8974.001.json`. If that's the case, this could be rewritten as `report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json`.

Thoughts?

Reply:
Done.